app/lib/actions/katello/host/hypervisors_update.rb

```diff
@@ -30,8 +32,16 @@ def run
               update_facts(uuid, host)
             end
           end
+
+          @duplicate_uuid_hypervisors.each do |hypervisor, existing_host|
```

Comment:
This will only surface one error, but this is a problem that should only affect a small percentage of users, so I opted to go this route rather than trying to cobble together a very large error message

Reply:
Sounds like a good plan +1